pontoon/base/static/js/main.js

```diff
@@ -479,18 +479,19 @@ $(function() {
 
   // Menu sort
   $('.menu .sort span').click(function (e) {
-    function val(index, el) {
-      if (index !== 2) {
-        return $(el).find('span:eq(' + index + ')').html();
+    function getString(el) {
+      return $(el).find('span:eq(' + index + ')').html();
+    }
 
-      } else {
-        if (!$(el).find('.chart').length) {
-          return 0;
-        }
-        var chart = $(el).find('.chart').data('chart'),
-            data = JSON.parse(chart.replace(/'/g, "\""));
-        return data.approved/data.total;
+    function get(el, type) {
+      if (!$(el).find('.chart').length) {
+        return 0;
       }
+
+      var chart = $(el).find('.chart').data('chart'),
+          data = JSON.parse(chart.replace(/'/g, "\""));
+
+      return data[type]/data.total;
```

Comment:
Instead of just returning one index from the data, why not return both the `approved` and `translated` percentages? Then you only call `get` once per list item and save on some math and JSON parsing.